src/appengine/handlers/cron/grouper.py

```diff
@@ -36,10 +36,12 @@ class TestcaseAttributes(object):
 
   __slots__ = ('id', 'is_leader', 'issue_id') + FORWARDED_ATTRIBUTES
 
-  def __init__(self, testcase_id):
+  def __init__(self, testcase_id, job_type=None, sec_flag=None):
```

Comment:
Rather than add these properties in the constructor, can we just set them later when we need them? 

No other properties get this, and there's no reason these two ones we added get this special treatment :)

Reply:
Done.